### ISSUE TITLE
Incorrect Copyright Date in Footer

### ISSUE THREAD

**Issue by animeshchaudhri** [AOSSIE-Org/website#16]
The footer currently displays an inaccurate copyright date, likely due to oversight or lack of regular updates.

**Comment by attoubo** [comment]
Hello @animeshchaudhri, i'm new to open source, can I work on it please ?

**Comment by animeshchaudhri** [comment]
Hello, Sir
I am writing to bring to your attention a minor but important issue regarding the copyright date displayed in the footer of the AOSSIE website. Despite previously raising the issue, it seems to have gone unaddressed. 
As a solution, I suggest implementing a dynamic date feature that will automatically update the copyright year, ensuring accuracy without the need for manual intervention in the future.
https://github.com/AOSSIE-Org/website/issues/16
@ceilican

**Comment by attoubo** [comment]
@animeshchaudhri did you assign me ???

**Comment by ceilican** [comment]
@animeshchaudhri , I agree. A dynamic update would be awesome. I assigned this issue to you. Looking forward to the PR with your solution.

**Comment by ceilican** [comment]
@attoubo , I think @animeshchaudhri proposed this issue because he wanted to work on it himself. Therefore, I have assigned it to him.

But you are welcome to look for other issues in our repos or propose your own issues/tasks. Looking forward to your contributions as well.

**Comment by attoubo** [comment]
Ok, sir @ceilican , I understand. Thank you

**Comment by AnniThing** [comment]
Hi @animeshchaudhri, are you still working on this issue?
because it's still shows open, so can i have it assigned to me
Thanks

**Comment by dipansrimany2006** [comment]
Hi, @animeshchaudhri I thought of a solution for this issue, and I can tell you about that. Can you assign me to this issue?

**Comment by govindsingh3477** [comment]
Still the issue is open , can i contribute in it

**Comment by siddharthbaleja7** [comment]
@ceilican if this issue is open, can you assign it to me?
